SimpleWAWS/Global.asax.cs

```diff
@@ -189,9 +207,9 @@ protected void Application_Error(object sender, EventArgs e)
             {
                 Exception ex = Server.GetLastError();
 
-                if (Response.StatusCode >= 500)
+                if (Response.StatusCode >= 400)
                 {
-                    SimpleTrace.Diagnostics.Error(ex, "Exception from Application_Error");
+                    AppInsights.TelemetryClient.TrackException(ex);
```

Comment:
Actually I don't think Application_Error works for WebAPI, I wasn't seeing any traces from this, and never fixed it. According to AppInsight docs it should be done in an [`IExceptionLogger`?](https://github.com/projectkudu/AzureFunctionsPortal/blob/dev/AzureFunctions/Global.asax.cs#L39)